modules/azure/storage.go

```diff
@@ -191,6 +192,7 @@ func GetStorageAccountClientE(subscriptionID string) (*storage.AccountsClient, e
 }
 
 // GetStorageBlobContainerClientE creates a storage container client.
```

Comment:
remove the method GetStorageBlobContainerClientE  as it is no longer used

Reply:
I mentioned this somewhere, but I can't find where I said this, so reiterating it here:

Since this is a public method, removing it means we will have to cut a backwards incompatible release. It would be a bit of a pain for us to release a backwards incompatible release for each of these changes. I think it would be better to keep these in so that we can merge all of these changes as patch releases, and then at the very end once all the files are updated, we can have one big PR to remove all the `Get.*Client` functions and cut one backwards incompatible release.

Reply:
sounds good @yorinasub17 , I approved it and it is ready for  review  @gruntwork-terratest-terragrunt